android-app/app/build.gradle

```diff
@@ -52,7 +52,7 @@ android {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'com.google.android.material:material:1.8.0'
+    implementation 'com.google.android.material:material:1.7.0'
```

Comment:
```suggestion
    implementation 'com.google.android.material:material:1.7.0' // cannot update to anything > 1.7.0 yet because of missing stable Android Studio updates https://github.com/material-components/material-components-android/issues/3186
```